src/test/groovy/wooga/gradle/release/ReleasePluginSpec.groovy

```diff
@@ -310,9 +307,10 @@ class ReleasePluginSpec extends ProjectSpec {
         _               | 5        | "snapshot" | _       | "hotfix-check"  | "1.0.1-branchHotfixCheck00005"
         _               | 6        | "snapshot" | _       | "fix-check"     | "1.0.1-branchFixCheck00006"
         _               | 7        | "snapshot" | _       | "PR-22"         | "1.0.1-branchPRTwoTwo00007"
-        _               | 8        | "snapshot" | _       | "fix/a_-bug"    | "1.0.1-branchFixABug00008"
-        _               | 9        | "snapshot" | _       | "fix/-_.bug"    | "1.0.1-branchFixDotbug00009"
-        _               | 10       | "snapshot" | _       | "fix/-.-.-bug"  | "1.0.1-branchFixDotDotBug00010"
+        // TODO: Not supported by the atlas-version plugin currently
+//        _               | 8        | "snapshot" | _       | "fix/a_-bug"    | "1.0.1-branchFixABug00008"
+//        _               | 9        | "snapshot" | _       | "fix/-_.bug"    | "1.0.1-branchFixDotbug00009"
+//        _               | 10       | "snapshot" | _       | "fix/-.-.-bug"  | "1.0.1-branchFixDotDotBug00010"
```

Comment:
please remove and or open an issue at the atlas-version plugin to check if support can be added.

Reply:
An issue was opened.